api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,156 @@ module: excel_exporter
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports grid data to Excel.
 
 ##### return: Promise<CellRange>
-<!-- Description goes here -->
+A <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">Promise</a> that is resolved with an object that contains the coordinates of the first and last cells. 
 
 ##### param(options): ExportDataGridProps
-<!-- Description goes here -->
+Exported **DataGrid** settings.
+
+---     
+
+Note that **exportDataGrid(options)** utilizes <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> API to implement export. 
```

Comment:
It is unclear what a developer should do with that information. Are we saying that it is required to reference/import ExcelJS (and the FileSaver, too)?

Reply:
Not only that, but we also remind our customers that we rely on the third-party library, and should read more about this API in the ExcelJS docs.